acvm_js/README.md

```diff
@@ -0,0 +1,17 @@
+# acvm_js
+
+The `acvm_js` package enables users to execute an ACIR program, i.e. generating an initial witness from a set of inputs and calculating a partial witness. This partial witness can then be used to create a proof of execution using an ACVM backend.
+
+## Dependencies
+
+In order to build the wasm package, the following must be installed:
+
+- [jq](https://github.com/stedolan/jq)
+
+## Build
+
+The wasm package can be built using the command below:
+
+```bash
+./build.sh
+```
```

Comment:
This isn't correct.

Reply:
Please expand, this is still valid when in context of that current folder.

Reply:
If I run this then I get

```
tom at tom-desktop in ~/Programming/acvm/acvm_js on combined_history-acvm_js [$]
$ ./build.sh
Will install package to /home/tom/Programming/acvm/outputs/out (defined outside installPhase.sh script)
   Compiling acvm_js v0.21.0 (/home/tom/Programming/acvm/acvm_js)
    Finished release [optimized] target(s) in 1.24s
error: failed reading '/home/tom/Programming/acvm/acvm_js/target/wasm32-unknown-unknown/release/acvm_js.wasm'

Caused by:
    No such file or directory (os error 2)
Command 'wasm-bindgen /home/tom/Programming/acvm/acvm_js/target/wasm32-unknown-unknown/release/acvm_js.wasm --out-dir /home/tom/Programming/acvm/acvm_js/pkg/nodejs/ --typescript --target nodejs' failed with exit code 1
Command './buildPhaseCargoCommand.sh' failed with exit code 1
```

If I try and run this from the monorepo root, I instead get

```
tom at tom-desktop in ~/Programming/acvm on combined_history-acvm_js [$]
$ ./acvm_js/build.sh
Will install package to /home/tom/Programming/acvm/outputs/out (defined outside installPhase.sh script)
./acvm_js/build.sh: line 17: ./buildPhaseCargoCommand.sh: No such file or directory
Command './buildPhaseCargoCommand.sh' failed with exit code 127
```

It seems that the only way to build acvm_js is through `nix build .` from the monorepo root.

Reply:
Not a blocker btw.